Cargo.toml

```diff
@@ -20,8 +20,8 @@ include = [
 ]
 
 [dependencies]
-nom = "^3"
-rusticata-macros = "0.4.0"
+nom = "~4.0"
```

Comment:
`~4.0` will prevent upgrading to 4.1, if I'm correct ? Caret dependency should be fine here, I think